openfisca_us/parameters/contrib/congress/senate/democrats/inflation_reduction_act/electric_vehicle_credit/new/eligibility/msrp_limit.yaml

```diff
@@ -0,0 +1,18 @@
+description: MSRP limit to qualify for the new electric vehicle credit under the Inflation Reduction Act.
+metadata:
+  reference:
+    - title: Inflation Reduction Act
+      href: https://www.democrats.senate.gov/imo/media/doc/inflation_reduction_act_of_2022.pdf#page=377
+  label: MSRP limit for new EV credit under the IRA
+  unit: currency-USD
```

Comment:
@nikhilwoodruff what period should I put here? It currently doesn't display right in PolicyEngine.

Reply:
Nothing for now - will fix on in the policyengine repo